bundles/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/dto/UniFiClient.java

```diff
@@ -83,7 +93,7 @@ public Integer getUptime() {
         return uptime;
     }
 
-    public Calendar getLastSeen() {
+    public ZonedDateTime getLastSeen() {
```

Comment:
`Instant` seems more appropriate here. UniFiTimestampDeserializer originally applies system default time-zone to an instant, and when using it to create a DateTimeType system default time-zone is also applied. So having the time-zone internally seems unused and overly complicated.